app/src/main/java/de/xikolo/controllers/base/CollapsingToolbarViewModelActivity.kt

```diff
@@ -0,0 +1,79 @@
+package de.xikolo.controllers.base
+
+import android.os.Bundle
+import android.view.View
+import android.widget.ImageView
+import androidx.coordinatorlayout.widget.CoordinatorLayout
+import butterknife.BindView
+import com.google.android.material.appbar.AppBarLayout
+import com.google.android.material.appbar.CollapsingToolbarLayout
+import de.xikolo.R
+import de.xikolo.R.id.appbar
+import de.xikolo.R.id.collapsing_toolbar
+import de.xikolo.viewmodels.base.BaseViewModel
+
+abstract class CollapsingToolbarViewModelActivity<T : BaseViewModel> : ViewModelActivity<T>() {
+
+    companion object {
+        val TAG: String = CollapsingToolbarViewModelActivity::class.java.simpleName
+    }
+
+    @BindView(R.id.toolbar_image)
+    lateinit var imageView: ImageView
+
+    @BindView(appbar)
+    lateinit var appBarLayout: AppBarLayout
+
+    @BindView(collapsing_toolbar)
```

Comment:
`appbar` and `collapsing_toolbar` don't start with `R.id`, so where do they come from?

Reply:
somehow these landed in the imports. Fixed that.